### ISSUE TITLE
'Upload' button for corpus importing

### ISSUE THREAD

**Issue by akutuzov** [jonorthwash/ud-annotatrix#263]
Currently, to load an existing corpus, after choosing the file one has to click the 'close' button: 
![screenshot_20180123_153928](https://user-images.githubusercontent.com/3366913/35282300-6dd4728e-0055-11e8-8bd3-4645206a03e5.png)

This is very non-intuitive; would be much better to do actual  importing after click on the 'Upload' button, while the 'close' button should function as 'Cancel'.

**Comment by ftyers** [comment]
@akutuzov is right, this is not optimal. The behaviour should be as follows: (1) upload should only happen when the user clicks the upload button. (2) during upload the "Close" button should be "Cancel", (3) when the upload is done, the "Close" button should be "Close".

**Comment by maryszmary** [comment]
@ftyers, why not just close the modal window automatically after the upload is done? I think, it's non-intuitive for users to have to press one more button when the corpus is already uploaded.

**Comment by ftyers** [comment]
@maryszmary one reason might be in case the user wants to see how much space they used up. But I don't have a strong opinion about that either way. But I do have a strong feeling about (1).

**Comment by jonorthwash** [comment]
No, "close" should always be "cancel", so that you can decide *not* to upload the corpus.

I think we never resolved the issue of adding to versus replacing the currently loaded corpus.  (Or with backend mode, the additional option of adding a new corpus.)  @gstark0, were you working on this?

**Comment by keggsmurph21** [comment]
new behavior:

click "Browse" to choose a file
click "Upload" to send to server && redirect to edit it
click "Close" to cancel (will change this text to "Cancel") and exit the modal

**Comment by keggsmurph21** [comment]
check out [this example](https://imgur.com/FoYvO3L), let me know if there's something else that needs to be done

**Comment by jonorthwash** [comment]
That looks great.  I assume it gets sent to the server too?

**Comment by jonorthwash** [comment]
Also, I liked the size report from before. Can we have that back? I think a notification about the limit of local storage if server isn't available would be good too (iirc, we had that before). Does local storage limit corpus size even with the back end running?
